packages/interaction-kit/src/interactions/application-commands/slash-command-interaction.ts

```diff
@@ -35,7 +34,7 @@ export default class SlashCommandInteraction extends ApplicationCommandInteracti
 		return new Proxy(
 			{},
 			{
-				get: (target, property): OptionType | null =>
+				get: (target, property): ApplicationCommandInteractionDataOption | null =>
```

Comment:
This should fix #55.

Reply:
ill trust you on this and just merge lol

Reply:
wait you already did welp